daemon/logger/journald/read.go

```diff
@@ -101,21 +101,23 @@ package journald
 //	}
 //	return rc;
 //}
-//static int wait_for_data_or_close(sd_journal *j, int pipefd)
+//static int wait_for_data_cancelable(sd_journal *j, int pipefd)
```

Comment:
While cleaning up, it might be better to move this to a separate file so it can be read more easily.